src/main/java/dev/jbang/cli/Completion.java

```diff
@@ -27,4 +55,256 @@ public int completion() throws IOException {
 		out.flush();
 		return EXIT_OK;
 	}
+
+	// copied from picocli fish PR https://github.com/remkop/picocli/pull/2463
```

Comment:
So why not wait for that? It must be because you personally want to use it because otherwise I'd say : why waste the time :-)